pegasus/sites.v3/code.org/public/educate/weblab.html

```diff
@@ -0,0 +1,222 @@
+<div id="main-content" class="container">
```

Comment:
This should be a .haml file, rather than .html.  It might also make sense to put this in dashboard, served at https://studio.code.org.